src/printer/manager.py

```
@@ -1,17 +1,22 @@
 import os
```

Ruff (0.11.9)

`os` imported but unused

Remove unused import: `os`

(F401)